javalib/src/main/scala/java/lang/System.scala

```diff
@@ -146,6 +158,11 @@ object System {
     }
   }
 
+  private def getUserName(): Option[String] = {
+    val key = if (isWindows) "%USERNAME%" else "USER"
```

Comment:
I was thinking it may be better to use a C function call like maybe - https://pubs.opengroup.org/onlinepubs/007904875/functions/getlogin.html

Of course that is probably harder and I am not sure how to do the windows version.

Otherwise it can be spoofed?
```sh
% export USER="foo"
% echo $USER
foo
```